packages/react/src/components/form/Radio/Radio.mdx

```diff
@@ -34,12 +35,26 @@ import { Radio } from '@digdir/designsystemet-react';
 
 <Canvas of={RadioStories.AriaLabel} />
 
-## `Fieldset`
+## `Fieldset` og `useRadioGroup`
 
-Bruk `Fieldset` for gruppering av alternativknapper.
+Bruk `Fieldset` og `useRadioGroup` for gruppering av alternativknapper.
 
 <Canvas of={RadioStories.Group} />
-<Controls of={RadioStories.Group} />
+
+Til `useRadioGroup` kan du sende inn:
+
+<ArgTypes of={useRadioGroupStories.UseRadioGroup} />
+
+I respons får du objektet:
+
+```ts
+{
+  getRadioProps: (value: string) => RadioProps, // Returnerer props du kan spre på <Radio /> eller <input type="radio" />
+  validationMessageProps: ValidationMessageProps, // Props du valgfritt spre på ValidationMessage for å koble feilmelding til radio gruppen
+  setValue: string, // Funksjon for å sette radio gruppens verdi
+  value: string, // Nåværende verdi for radio gruppen
+}
+```
```

Comment:
I think we should have an easy code example of this as well (```tsx), since the code from they story does not show how to use it:
![image](https://github.com/user-attachments/assets/c0116b00-a70f-4507-bebb-2225f7033f2e)

Reply:
Agreed, good point :)
Sad storybook is not able to show the code 1:1 as we write it

Reply:
Yeah, I'm also wondering if we should also have stories for these under utilities as well, since the documentation is getting pretty large, and the hook might be a bit hidden 🤔 
Can also do this later when we have a look at documentation as a whole

Reply:
Yeah, all the stories are using the hook, but "show code" does not show it even though it is coded this way :(